src/Metadata/AbstractOperation.php

```diff
@@ -0,0 +1,776 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Metadata;
+
+abstract class AbstractOperation
```

Comment:
```suggestion
abstract class Operation
```

The API will be nicer, and this is consistent with the newest coding convention used in Symfony.

Reply:
I know right, but we need to rename the current one then

Reply:
Not an issue as this was never released isn't it?